tensorflow_encrypted/protocol/pond.py

```diff
@@ -67,10 +91,10 @@ def define_constant(self, value: np.ndarray, apply_scaling: bool=True, name: Opt
         with tf.name_scope('constant{}'.format('-' + name if name else '')):
 
             with tf.device(self.server_0.device_name):
-                x_on_0 = BackingConstant.from_same(v)
+                x_on_0 = BackingConstant.from_same(v, M)
```

Comment:
let's find a way to avoid this as it'll be weird otherwise: sometimes `M` is respected by the backing tensor, sometimes not

Reply:
Down!